rust/xaynet-server/src/storage/redis.rs

```diff
@@ -258,7 +197,60 @@ impl Connection {
         Ok(seed_dict)
     }
 
+    /// Updates the [`SeedDict`] with the seeds from the given ['UpdateParticipantPublicKey'].
+    pub async fn update_seed_dict(
+        mut self,
+        update_pk: &UpdateParticipantPublicKey,
+        local_seed_dict: &LocalSeedDict,
+    ) -> RedisResult<SeedDictUpdate> {
+        let script = Script::new(
+            r#"
+                local update_pk = ARGV[1]
+
+                -- check if the local seed dict has the same length as the sum_dict
+
+                -- KEYS is a list (table) of key value pairs ([update_pk_1, seed_1, update_pk_2, seed_2])
```

Comment:
Where does `KEYS` come from?

Reply:
Sorry I found the answer in the redis documentation: https://redis.io/commands/eval